Azure/azure-cli
# [AKS] az aks update: Change --enable-aad argument to migrate a RBAC-enabled non-AAD cluster to a AKS-managed AAD cluster

**Description**  


This PR is to port https://github.com/Azure/azure-cli-extensions/pull/2017 from azure-cli-extension to azure-cli, which supports migrating a RBAC-enabled non-AAD cluster to a AKS-managed AAD cluster.

**Testing Guide**  


1. Create RBAC-enabled non-AAD cluster.

    ```
    az aks create -g MyResourceGroup -n MyManagedCluster 
    ```

2. Migrate the cluster to the new AKS-managed AAD experience.

    ```
    az aks update -g MyResourceGroup -n MyManagedCluster --enable-aad
    ```

---

This checklist is used to make sure that common guidelines for a pull request are followed.
